console/src/components/AppIconCell/AppIconCell.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import kind from '@enact/core/kind';
+import {Cell} from '@enact/ui/Layout';
+import LabeledIconButton from '@enact/agate/LabeledIconButton';
+
+import css from './AppIconCell.less';
+
+const AppIconCell = kind({
+	name: 'AppIconCell',
+
+	styles: {
+		css,
+		className: 'iconCell'
+	},
+
+	render: ({children, size, ...rest}) => (
+		<Cell component={LabeledIconButton} align="center" size={size} {...rest}>{children}</Cell>
```

Comment:
Since `size` is unaltered, we can omit `size={size}` from the props list, as it will just be handled naturally by the `{...rest}` bit.

Alternatively, we could set a common-use default here, and since the `size` prop falls before the `{...rest}` we can override it in any component.

For example:
```jsx
<Cell component={LabeledIconButton} align="center" size={180} {...rest}>{children}</Cell>
```
The above will render as 180px wide, unless a consumer of this component provides a new (replacement) `size` prop.
```jsx
// This renders as 180px
<AppIconCell icon="compass">Navigation</AppIconCell>
// This renders at 40%
<AppIconCell size="40%" icon="compass">Navigation</AppIconCell>
```

The `align="center"` is probably not intended here. The `align` prop on a `Cell` locally overrides the parent Layout's (Row/Column) alignment. By having it here, the Row or Column's alignment is canceled out and nullified.